modules/administration-guide/partials/proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc

```diff
@@ -1,114 +1,83 @@
-+
-.Prometheus configuration
 ====
-[source,yaml,subs="+quotes,+attributes,+macros"]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: prometheus-config
-  namespace: monitoring
-data:
-  prometheus.yml: |-
-      global:
-        scrape_interval: 5s <1>
-        evaluation_interval: 5s <2>
-      scrape_configs: <3>
-        - job_name: 'DevWorkspace'
-          scheme: https
-          authorization:
-            type: Bearer
-            credentials_file: '/var/run/secrets/kubernetes.io/serviceaccount/token'
-          tls_config:
-            insecure_skip_verify: true
-          static_configs:
-            - targets: ['devworkspace-controller-metrics.__<DWO_{orch-namespace}>__:8443'] <4>
-        - job_name: 'DevWorkspace webhooks'
-          scheme: https
-          authorization:
-            type: Bearer
-            credentials_file: '/var/run/secrets/kubernetes.io/serviceaccount/token'
-          tls_config:
-            insecure_skip_verify: true
-          static_configs:
-            - targets: ['devworkspace-webhookserver.__<DWO_{orch-namespace}>__:9443'] <5>
-----
-<1> The rate at which a target is scraped.
-<2> The rate at which the recording and alerting rules are re-checked.
-<3> The resources that Prometheus monitors. In the default configuration, two jobs, `DevWorkspace` and `DevWorkspace webhooks`, scrape the time series data exposed by the `devworkspace-controller-metrics` and `devworkspace-webhookserver` Services.
-<4> The scrape target for the metrics from port `8443`. Replace `__<DWO_{orch-namespace}>__` with the {orch-namespace} where the `devworkspace-controller-metrics` `Service` is located.
-<5> The scrape target for the metrics from port `9443`. Replace `__<DWO_{orch-namespace}>__` with the {orch-namespace} where the `devworkspace-webhookserver` `Service` is located.
-====
-
-. Scale the `Prometheus` Deployment down and up to read the updated ConfigMap from the previous step.
-+
 [source,terminal,subs="+attributes,quotes"]
 ----
-$ {orch-cli} scale --replicas=0 deployment/prometheus -n monitoring && {orch-cli} scale --replicas=1 deployment/prometheus -n monitoring
+oc label namespace {prod-namespace} openshift.io/cluster-monitoring=true
 ----
+====
 
 .Verification
 
-. Use port forwarding to access the `Prometheus` Service locally:
-+
-[source,terminal,subs="+attributes,quotes"]
-----
-$ {orch-cli} port-forward svc/prometheus 9090:9090 -n monitoring
-----
-. Verify that all targets are up by viewing the targets endpoint at `localhost:9090/targets`.
-. Use the Prometheus console to view and query metrics:
-** View metrics at `localhost:9090/metrics`.
-** Query metrics from `localhost:9090/graph`.
+. For a fresh installation of {prod-short}, generate metrics by creating a {prod-short} workspace from the Dashboard.
+
+. In the *Administrator* view of the OpenShift web console, navigate to *Observe* -> *Metrics* from the sidebar menu on the left.
```

Comment:
Our style guide recommends avoiding positional references in the UI like "the sidebar menu on the left":
```suggestion
. In the *Administrator* view of the OpenShift web console, go to *Observe* -> *Metrics*.
```